ext/reflection/php_reflection.c

```diff
@@ -3778,7 +3783,7 @@ ZEND_METHOD(reflection_class, getStaticProperties)
 	}
 
 	array_init(return_value);
-	add_class_vars(ce, 1, return_value);
+	add_class_vars(ce, 1, 0, return_value);
```

Comment:
Please inline (duplicate) this case instead.

Reply:
Done. Would it possible to simplify that further?